src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.net46.cs

```diff
@@ -8,17 +8,17 @@ namespace System.IO
 {
     public static class FileSystemAclExtensions
     {
-        public static DirectorySecurity GetAccessControl(DirectoryInfo directoryInfo)
+        public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo)
         {
             return directoryInfo.GetAccessControl();
         }
 
-        public static DirectorySecurity GetAccessControl(DirectoryInfo directoryInfo, AccessControlSections includeSections)
+        public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo, AccessControlSections includeSections)
```

Comment:
Does `this` already exist in the ref assembly on all of these, or does it need to be added there, too?